itur/validation/validation_scripts/Lred_database_chekcer.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+"""
+Creates 2D lists of the L_red database files for P.840 revisions & and 8.
+ 
+This will allow the user to compare the differences between revisions using
+equality commands in the console after running through the script. 
+
+Variables Names:
+    Revision 7:
+        rev8_01List = L_red database in revision 7 with probability 0.1
+        rev8_1List = L_red database in revision 7 with probability 1
+        
+    Revision 8:
+        rev8_01List = L_red database in revision 8 with probability 0.1
+        rev8_1List = L_red database in revision 8 with probability 1
+
+Created on Thu Jun 25 15:00:36 2020
+
+@author: MAW32652
+"""
+import csv
+
+### Creating the revision 7 variables
+
+with open('C:/Users/MAW32652/Desktop/Revision 7/01test7.csv', newline='') as rev7_01:
```

Comment:
There cannot be hardcoded absolute paths in the commit. It will break when run on a different machine.